egs/wsj/s5/utils/reverse_lm.sh

```diff
@@ -38,25 +38,13 @@ mkdir -p $outdir
 for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
   cp -r $langdir/$f $outdir
```

Comment:
I think if you are going to change the reverse_lm.sh script like this you should try to run that stuff- it's in the WSJ example.  Otherwise leave it alone- it's never really used these days.  For instance, I'm not sure if the forbidden combinations of <s> and </s> would interact badly with the reverse_arpa script.